command/CommandController.js

```diff
@@ -261,31 +261,95 @@ async function processArticleRequest(
   }
   logger.debug('result.url is: ' + result.url);
 
-  if (endInstructions) {
-    let expireDate = new Date();
-    // Set the expire_date to 30 days ago as S3 deletes expired files
-    expireDate.setDate(expireDate.getDate() - 30);
-    if (new Date(endInstructionsData.date) < expireDate) {
-      if (process.env.META_VOICE) {
-        endInstructionsData.url = await buildAudioFromText(
-          endInstructionsData.text,
-          process.env.META_VOICE
-        );
-      } else {
-        endInstructionsData.url = await buildAudioFromText(
-          endInstructionsData.text
-        );
-      }
-      endInstructionsData.date = Date.now();
-    }
-    result.instructions_url = endInstructionsData.url;
+  if (metaAudioRequested) {
+    let metaAudio = await generateMetaAudio(result, summaryOnly);
+
+    result.instructions_url = metaAudio.instructions_url;
+    result.intro_url = metaAudio.intro_url;
+    result.outro_url = metaAudio.outro_url;
   }
   // Initially set offset to 0 (overwrite later if necessary)
   result.offset_ms = 0;
 
   return result;
 }
 
+async function generateMetaAudio(data, summaryOnly) {
```

Comment:
Can you add a comment ahead of this function describing it's purpose and what it covers?

Reply:
Let's also include in this comment when/how you are storing the intro/outro pieces.